### ISSUE TITLE
Missing in Release Notes for 6.24 

### ISSUE THREAD

**Issue by lmoneta** [root-project/root#7858]
List of relevant PR missing in the release notes for 6.24

- XROOTD  #7380 
- TMVA : optimizer options : #7318
- ChangeLabel: #7294
- Buffer Merger improvements : #7286
- Eve7: The first implementation of log view #7270
- TMVA: Update Keras tutorials using tensorflow.keras #7255
- builtins:  Zstd 1.4.8 #7214
- RooFit: warning message for RooAddPdf without norm set: #7129 
- RooFit: Improve plotting of RooBinSamplingPdf. #7110
- Hist: Fixes and improvements for the TKDE class #7096
- Fix a memory leak in TF1::Copy #7035
- Fix for new version of cudnn version 8 #7003
- Minuit2: Use Numerical gradient as input to the seed generator when using external gradients in Minuit2 #7001
- TF1: Enable using Automatic differentiation for computing Parameter Gradient in TF1 #6956
- Math: Fix correction factor for Confidence level in ROOT::Fit::FitResult #6902
- TMVA: Implement MethodCategory::GetMulticlassValues() #6886
- Hist: Add THistRange class defining a generic histogram range and its iterator #6795
- TMVA: Support  for Keras 2.4 (tf.keras) #6793
- Eve-7: Add example of calorimeter representation of REveDataCollection #6783
- Math: Fix BinData copy constructor #6776
- [tmva] Add possibility to make ROC curve for the training dataset from TMVA::Factory #6724
- Math: Changes in TRandom3::GetSeed  #6723
- Minuit2: new Minuit2 logging #6705
- ntuple] Several interface improvements (unique pointer, const) #6655
- Optimize TPrincipal::AddRow function #6630
